src/terminal/adapter/MouseInput.cpp

```diff
@@ -181,238 +176,239 @@ int MouseInput::s_WindowsButtonToXEncoding(const unsigned int uiButton,
+// Parameters:
+// - sCoordinateValue - the value to encode.
+// Return value:
+// - the encoded value.
+short MouseInput::s_EncodeDefaultCoordinate(const short sCoordinateValue) noexcept
+{
+    return sCoordinateValue + 32;
+}
+
+#pragma warning(pop)
+
 // Routine Description:
 // - Attempt to handle the given mouse coordinates and windows button as a VT-style mouse event.
 //     If the event should be transmitted in the selected mouse mode, then we'll try and
 //     encode the event according to the rules of the selected ExtendedMode, and insert those characters into the input buffer.
 // Parameters:
-// - coordMousePosition - The windows coordinates (top,left = 0,0) of the mouse event
-// - uiButton - the message to decode.
-// - sModifierKeystate - the modifier keys pressed with this button
-// - sWheelDelta - the amount that the scroll wheel changed (should be 0 unless uiButton is a WM_MOUSE*WHEEL)
+// - position - The windows coordinates (top,left = 0,0) of the mouse event
+// - button - the message to decode.
+// - modifierKeyState - the modifier keys pressed with this button
+// - delta - the amount that the scroll wheel changed (should be 0 unless button is a WM_MOUSE*WHEEL)
 // Return value:
 // - true if the event was handled and we should stop event propagation to the default window handler.
-bool MouseInput::HandleMouse(const COORD coordMousePosition,
-                             const unsigned int uiButton,
-                             const short sModifierKeystate,
-                             const short sWheelDelta)
+bool MouseInput::HandleMouse(const COORD position,
+                             const unsigned int button,
+                             const short modifierKeyState,
+                             const short delta)
 {
-    bool fSuccess = false;
-    if (_ShouldSendAlternateScroll(uiButton, sWheelDelta))
+    bool success = false;
+    if (_ShouldSendAlternateScroll(button, delta))
     {
-        fSuccess = _SendAlternateScroll(sWheelDelta);
+        success = _SendAlternateScroll(delta);
     }
     else
     {
-        fSuccess = (_TrackingMode != TrackingMode::None);
-        if (fSuccess)
+        success = (_trackingMode != TrackingMode::None);
+        if (success)
         {
-            // fIsHover is only true for WM_MOUSEMOVE events
-            const bool fIsHover = s_IsHoverMsg(uiButton);
-            const bool fIsButton = s_IsButtonMsg(uiButton);
+            // isHover is only true for WM_MOUSEMOVE events
+            const bool isHover = s_IsHoverMsg(button);
+            const bool isButton = s_IsButtonMsg(button);
 
-            const bool fSameCoord = (coordMousePosition.X == _coordLastPos.X) &&
-                                    (coordMousePosition.Y == _coordLastPos.Y) &&
-                                    (_lastButton == uiButton);
+            const bool sameCoord = (position.X == _lastPos.X) &&
+                                   (position.Y == _lastPos.Y) &&

```

Comment:
aside: i am surprised we don't have an `operator==(COORD,COORD)` that would let us do this more.. succinctly.

Reply:
Maybe I'll implement one.

Reply:
Nah, gonna save this idea for til::coordinate